fastplotlib/layouts/_gridplot.py

```diff
@@ -34,14 +34,14 @@ def to_array(a) -> np.ndarray:
 
 class GridPlot(RecordMixin):
     def __init__(
-        self,
-        shape: Tuple[int, int],
-        cameras: Union[np.ndarray, str] = "2d",
-        controllers: Union[np.ndarray, str] = None,
-        canvas: Union[str, WgpuCanvas, pygfx.Texture] = None,
-        renderer: pygfx.WgpuRenderer = None,
-        size: Tuple[int, int] = (500, 300),
-        **kwargs,
+            self,
+            shape: Tuple[int, int],
+            cameras: Union[np.ndarray, str] = "2d",
+            controllers: Union[np.ndarray, str] = None,
+            canvas: Union[str, WgpuCanvas, pygfx.Texture] = None,
+            renderer: pygfx.WgpuRenderer = None,
+            size: Tuple[int, int] = (500, 300),
+            **kwargs,
```

Comment:
don't change the spacing, we're trying to figure out a linter to use but meanwhile try to keep it the same

Reply:
Ruff!

Reply:
Does it have an option to set code lines length but allow longer docstring length? Large docstrings are useful for tables!

Reply:
No clue, but ruff is so fast that fastplotlib must use it 😉